CONTRIBUTING.md

```diff
@@ -68,7 +68,7 @@ By making a contribution to this project, I certify that:
     consistent with this project or the open source license(s)
     involved.
  ```
-We require that every contribution to OpenSearch-SDK is signed with a Developer Certificate of Origin. Additionally, please use your real name. We do not accept anonymous contributors nor those utilizing pseudonyms.
+We require that every contribution to OpenSearch-SDK-JAVA is signed with a Developer Certificate of Origin. Additionally, please use your real name. We do not accept anonymous contributors nor those utilizing pseudonyms.
```

Comment:
"Developer Certificate of Origin" isn't clear to new contributors.  This needs to be linked to better documentation.

I'd suggest linking to the main project `CONTRIBUTING` doc:
https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin